types/underscore/index.d.ts

```diff
@@ -5334,8 +5242,7 @@ declare module _ {
          * @param propertyName The name of a specific property to retrieve from all items.
          * @returns The set of values for the specified property for each item in the collection in a chain wrapper.
          **/
-        pluck<K extends keyof T>(propertyName: K): _Chain<T[K], T[K][]>;
-        pluck(propertyName: string): _Chain<any, any[]>;
+        pluck<K extends KeyOrPropertyName<T>>(propertyName: K): _Chain<PropertyTypeOrAny<T, K>, PropertyTypeOrAny<T, K>[]>;
```

Comment:
This line is a bit long, maybe break the return type?

Reply:
I made this a multi-line function definition and moved the return type to its own line to break down this line and adjusted other functions to match.